src/mon/mon_types.h

```diff
@@ -24,16 +24,20 @@
 #include "common/bit_str.h"
 #include "common/ceph_releases.h"
 
-#define PAXOS_MDSMAP     0
-#define PAXOS_OSDMAP     1
-#define PAXOS_LOG        2
-#define PAXOS_MONMAP     3
-#define PAXOS_AUTH       4
-#define PAXOS_MGR        5
-#define PAXOS_MGRSTAT    6
-#define PAXOS_HEALTH     7
-#define PAXOS_CONFIG     8
-#define PAXOS_NUM        9
+// use as paxos_service index
+enum paxos_svc_type : int {
```

Comment:
```suggestion
enum {
```

Reply:
nit, the title of "enumerate paxos service type for scability" does not look quite right. IMHO, better off put something like

> mon/mon_types.h: define paxos services indexes using enum
>
> simpler this way

Reply:
Done